test/mbr_update_test.rb

```diff
@@ -28,6 +31,16 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
+      end
+
+      it "does nothing except returning true" do
+        expect(subject.run).to eq true
```

Comment:
> this mean, that for any new feature I need to verify that it really do nothing, that is a bit overkill for me.

Yes, and that will remind you of the "# FIXME: make it single responsibility class" ;-)

Since the tests already do contain many cases of `expect(...).to_not receive(...)` I think it is fair to add such expectations for the s390 case. Whether they are bundled inside a single "architecture" context or dispersed among the "responsibility" contexts, I don't care much.